vars/dockerBuildArgs.groovy

```diff
@@ -48,27 +100,46 @@ String call(Map config = [:]) {
       ret_str += " --build-arg CACHEBUST=${currentBuild.startTimeInMillis}"
       ret_str += " --build-arg CB0=" + current_time.get(Calendar.WEEK_OF_YEAR)
     }
-
     if (add_repos) {
-      if (env.REPOSITORY_URL) {
-        ret_str += ' --build-arg REPO_URL=' + env.REPOSITORY_URL
-      }
-      if (env.DAOS_STACK_EL_7_LOCAL_REPO) {
-        ret_str += ' --build-arg REPO_EL7=' + env.DAOS_STACK_EL_7_LOCAL_REPO
+      String repo_name = null
+      if (stage_info['target'] == 'centos7') {
+        repo_alias = 'EL_7'
+        repo_name = env["DAOS_STACK_${repo_alias}${repo_mod}_${repo_type}_REPO"]
+        if (repo_name) {
+          ret_str += ' --build-arg REPO_EL7=' + repo_name
+        }
       }
-      if (env.DAOS_STACK_EL_8_LOCAL_REPO) {
-        ret_str += ' --build-arg REPO_EL8=' + env.DAOS_STACK_EL_8_LOCAL_REPO
+      if (stage_info['target'] == 'centos8') {
+        repo_alias = 'EL_8'
+        repo_name = env["DAOS_STACK_${repo_alias}${repo_mod}_${repo_type}_REPO"]
+        if (repo_name) {
+          ret_str += ' --build-arg REPO_EL8=' + repo_name
+        }
       }
-      if (env.DAOS_STACK_LEAP_15_LOCAL_REPO) {
-        ret_str += ' --build-arg REPO_LOCAL_LEAP15=' +
-                   env.DAOS_STACK_LEAP_15_LOCAL_REPO
+      if (stage_info['target'] == 'leap15') {
+        repo_alias = 'LEAP_15'
+        repo_name = env["DAOS_STACK_${repo_alias}${repo_mod}_${repo_type}_REPO"]
+        if (repo_name) {
+          ret_str += ' --build-arg REPO_EL7=' + repo_name
+        }
+        if (repo_type == 'LOCAL') {
+         if (env.DAOS_STACK_LEAP_15_GROUP_REPO) {
```

Comment:
Why is there (only) group repo support for Leap 15?

Reply:
We are going from having one local repo for locally built artifacts and a group repo for the OS artifacts to one group repo that has both.

By just using a group repo, it gets us around the issue that Nexus will not let us rename a local repo if we want save it and start using a different repo.